src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidatorTest.java

```diff
@@ -292,11 +310,76 @@ void testErrorReportedWhenUnableToObtainAllTrustInfoFieldIsNull() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(2020,4, 10));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.of(2020,4, 11));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateBeforeCreationDate() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(2020,4, 11));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.of(2020,4, 10));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        String validationMessage = ValidationMessages.CEASED_DATE_BEFORE_CREATION_DATE_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateIsInTheFuture() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(2020,4, 11));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.now().plusDays(1));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        String validationMessage = ValidationMessages.DATE_NOT_IN_PAST_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateBeforeAnyIndividualsDateOfBirth() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        TrustIndividualDto individual = trustDataDtoList.get(0).getIndividuals().get(0);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(1990,4, 10));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.of(1990,4, 11));
+        individual.setDateOfBirth(LocalDate.of(1990,4, 12));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        String validationMessage = ValidationMessages.CEASED_DATE_BEFORE_INDIVIDUALS_DATE_OF_BRITH_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
```

Comment:
I can't see any tests for when trust is being ceased but date is after the individual DOB and no error is reported. Have I missed that? Also need to ensure that we test this scenario (another test) when there are multiple individuals and all their dates are checked.

Reply:
Added further coverage for these scenarios.